ccip-sdk/src/chain.ts

```diff
@@ -212,13 +228,34 @@ export abstract class Chain<F extends ChainFamily = ChainFamily> {
    *     want to assume those are optimization hints, instead of hard filters, and verify results
    *   - `page`: if provided, try to use this page/range for batches
    * @returns An async iterable iterator of logs.
+   *
+   * @remarks
+   * Behavior varies by chain family:
+   * - **EVM**: Full topic filtering, supports both startBlock and endBlock
+   * - **Solana**: Requires address (program ID), may have pagination limits
+   * - **Aptos**: Different log structure and filtering capabilities
```

Comment:
The behavior should be pretty consistent across chain families; everything but EVM requires address, and some methods accept some special params, but otherwise, they all should match the generic interface behavior described above